python/cudaq/kernel/ast_bridge.py

```diff
@@ -1639,6 +1655,29 @@ def process_potential_ptr_types(value):
             self.emitFatalError("unrecognized attribute {}".format(node.attr),
                                 node)
 
+    def find_unique_decorator_name(self, name):
+        mod = sys.modules[self.kernelModuleName]
```

Comment:
Is `self.kernelModuleName` a new property of `PyASTBridge`?

Reply:
This includes changes which will be used later.